### DISCUSSION TITLE
jspsych-survey-html-form max number of characters

### DISCUSSION THREAD

**Discussion by sacchilvia** [jspsych/jsPsych#1914] [Q&A] [answered]
Hi!
In my experiment, I am asking participants to memorize sequences of consonants and to report them after a while. To record what they remember I am using the jspsych-survey-html-form plugin. I would like to set a maximum number of characters that can be inserted, which is the easiest way to do so? 
I have tried to insert size="2" in the html parameter, but it just changes the size of the box.
I have also tried to use the jspsych-survey-text plugin and to change the column parameter, but again it just changes the size of the box.

Thanks a lot in advance!

Silvia

**Comment by becky-gilbert** [comment, marked as answer]
Hi @sacchilvia, I think the attribute you want is [`maxlength`](https://www.w3schools.com/tags/att_input_maxlength.asp).

**Comment by sacchilvia** [comment]
It works perfectly, thanks a lot!
